docs/reference/compatibility.md

```diff
@@ -68,6 +68,8 @@ ONNX Runtime supports all opsets from the latest released version of the [ONNX](
 
 | ONNX Runtime version | [ONNX version](https://github.com/onnx/onnx/blob/master/docs/Versioning.md) | ONNX opset version | ONNX ML opset version | ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
```

Comment:
https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes hasn't been updated since 1.9. 
https://github.com/microsoft/onnxruntime/blob/master/docs/Versioning.md#version-matrix refers to https://www.nuget.org/packages/Microsoft.AI.MachineLearning. Should the release-notes page be updated or should this table refer to the nuget page instead?

Reply:
We can submit a PR for https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes

Reply:
What do you think the best solution is?

Reply:
I will ask the winml folks. Ideally the release-notes are updated as they contain details of what changed.